src/widecharwidth/widechar_width.rs

```diff
@@ -23,37 +23,31 @@ type R = (u32, u32);
 #[derive(Copy, Clone, Debug, Eq, PartialEq)]
 #[repr(u8)]
 pub enum WcWidth {
-  /// The character is single-width
```

Comment:
`src/widecharwidth/widechar_width.rs` is generated code.

I think we should only format it once we can ensure that any future updates will be formatted as well.
Maybe exclude it until then.